packages/api/package.json

```diff
@@ -120,7 +120,7 @@
     "rewire": "^4.0.1",
     "sinon": "^7.1.1",
     "supertest": "^3.3.0",
-    "webpack": "~4.5.0",
+    "webpack": "^4.5.0",
```

Comment:
Just noticed this. Any reason for the change? 

Also, if we do need this, I would expect a corresponding change in the `package-lock.json` for this module

Reply:
oh... I don't know why this was changed. I'll try reverting and we'll see what breaks.

Reply:
Did it work when you removed it?